src/otx/recipe/detection/atss_mobilenetv2.yaml

```diff
@@ -20,19 +20,16 @@ scheduler:
       mode: max
       factor: 0.5
       patience: 5
-      monitor: val/map_50
+      monitor: val/f1-score
```

Comment:
Do we need to change the default metric type from `map_50` to `f1` in this codebase? It will make the regression tests problematic at this stage. In addition, I don't know why we should move to the less popular metric if we consider this is the default configuration for OTX standalone.

Reply:
OTX1.5 use mAP50 on validation and use F1 on test. And OTX1.5 report F1 on Geti. 
There's some suggestion that we should align metrics between validation and test. 
So I choose F1 as a validation and test metric, since it is hard to change Geti metric

Reply:
These two statements are different story each other: 1) Enable F1 evaluation metric for validation in Geti scenario and 2) Change the default evaluation metric in OTX standalone.

Reply:
I think there's two ways to report f1 score to Geti.
1. Make template for Geti, whose metric is f1
2. Geti use API of OTX so Geti can choose metric it want

What's your opinion?

Reply:
See https://teams.microsoft.com/l/message/19:meeting_MjNmM2E4ZmUtMDEyZi00M2Y5LWFjY2YtMDdmZjM2ZjUzZTRi@thread.v2/1707294490148?context=%7B%22contextType%22%3A%22chat%22%7D.
At this moment, the only one that we have to take an action is to enable using alternative metrics from configuration or API.